scripts/cmake/test/RunBenchmark.cmake

```diff
@@ -1,4 +1,4 @@
-set(BENCHMARK_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/benchmarks/${benchmarkDir})
+set(BENCHMARK_OUTPUT_DIRECTORY ${BENCHMARK_DIR}/${benchmarkDir})
```

Comment:
i see. then i will revert this change. is it still ok if I keep the change at line 50?